eitprocessing/eit_data/event.py

```diff
@@ -3,6 +3,8 @@
 
 @dataclass
 class Event:
+    """Single time point event registered during an EIT measurement on a Draeger machine."""
```

Comment:
```suggestion
    """Single time point event registered during an EIT measurement."""
```

Reply:
Currently, we are only importing it in the draeger module, which is why I put it like this. I think let's just remove the specification. We we will probably anyway ultimately settle on some unified nomenclature when we work out the `SparseData` functionality.